lib/SILOptimizer/PassManager/PassPipeline.cpp

```diff
@@ -482,10 +486,6 @@ static void addHighLevelFunctionPipeline(SILPassPipelinePlan &P) {
   // FIXME: update EagerSpecializer to be a function pass!
   P.addEagerSpecializer();
 
-  // We earlier eliminated ownership if we are not compiling the stdlib. Now
-  // handle the stdlib functions.
-  P.addNonTransparentFunctionOwnershipModelEliminator();
-
   addFunctionPasses(P, OptimizationLevelKind::HighLevel);
```

Comment:
Can you add a note here that we are lowering ownership in side addFunctionPasses. I am just trying to be very careful/explicit during the transition.

Reply:
Done